server.js

```diff
@@ -68,6 +68,22 @@ function getClasses(raceClasses) {
   return Promise.all(classesPromises);
 }
 
+app.get('/api/v1/faction/:name', async (request, response) => {
+  try {
+    const races = await database('races').where('faction', request.params.name).select();
```

Comment:
1. Get array with races that connected with the chosen faction from `races` table in DB